editor-service/src/main/java/com/qreal/robots/database/diagrams/model/DefaultDiagramNode.java

```diff
@@ -0,0 +1,113 @@
+package com.qreal.robots.database.diagrams.model;
+
+import com.qreal.robots.thrift.gen.TDefaultDiagramNode;
+
+import java.io.Serializable;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Diagram's node.
+ */
+public class DefaultDiagramNode implements Serializable {
```

Comment:
these classes are the same as in diagram DB service, but without annotatons. is there really no way to reuse the code?